TKeazirian.HTTPServer/Program.cs

```diff
@@ -1,11 +1,63 @@
-﻿namespace TKeazirian.HTTPServer;
+﻿using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace TKeazirian.HTTPServer;
+
 public static class Program
 {
-    private static void Main(string[] args)
+    public static string? IncomingData;
+
+    private static void StartListening()
     {
-        var g = new Greeting();
-        var hour = DateTime.Now.Hour;
-        var greeting = Greeting.GetGreeting(hour);
-        Console.WriteLine(greeting);
+        byte[] bytes = new byte[1024];
```

Comment:
What's this for? Why `1024`? Instantiating this so far from where it gets used makes it difficult for me to understand its purpose as the reader

Reply:
Good point.  From research, my understanding is it needs to indicate a certain number of bytes to receive?

Reply:
yes.  in C# an array is defined with a fixed size.  so this array will allow for up to 1024 bytes.